EnglishFilter/sections/general_extensions.txt

```diff
@@ -167,7 +167,7 @@ lookmovie.ag#%#//scriptlet("abort-on-property-write", "zfgproxyhttp")
 ! #%#//scriptlet("prevent-setInterval", "[$."), #%#//scriptlet("prevent-setTimeout", "[$."), #%#//scriptlet("prevent-addEventListener", "", "[$.")
 movies123.pk,manga-raw.club#%#//scriptlet("prevent-setInterval", "[$.")
 movies123.pk,123movieslinks.live,kickass.ws,kickass.love,kissanime.nz,animekisa.tv,repackov.com#%#//scriptlet("prevent-setTimeout", "[$.")
-movies123.pk,sportstreamtv.live,uwatchfree.tech,beinmatch.tv,fmovies.to,samehadaku.vip,kickass.ws,kickass.love,animafan.biz,manga-raw.club,kissanime.nz,animekisa.tv,repackov.com#%#//scriptlet("prevent-addEventListener", "", "[$.")
+123films.cc,movies123.pk,sportstreamtv.live,uwatchfree.tech,beinmatch.tv,fmovies.to,samehadaku.vip,kickass.ws,kickass.love,animafan.biz,manga-raw.club,kissanime.nz,animekisa.tv,repackov.com#%#//scriptlet("prevent-addEventListener", "", "[$.")
```

Comment:
we already have the rule for script: `#%#//scriptlet("abort-current-inline-script", "String.fromCharCode", "delete window")`

Reply:
Doesn't work. I reproduced pop-ups.

Reply:
@Alex-302 What should I do?

Reply:
Ok I will add it there.